mods/cnc/mod.yaml

```diff
@@ -29,13 +18,22 @@ Packages:
 	temperat.mix
 	winter.mix
 	desert.mix
-	snow.mix
 	~movies-gdi.mix
 	~movies-nod.mix
 	~movies.mix
 	~scores.mix
 	~scores2.mix
 	~transit.mix
+	./mods/cnc/bits
+	snow.mix
```

Comment:
Packages listed later override packages defined earlier.  The contents of the bits directory override the contents of the content mix files, so it is listed further down the list.  `snow.mix` is shipped in the bits directory, so must be referenced after bits (otherwise it wouldn't be found).